packages/headless/src/controllers/core/interactive-result/headless-core-interactive-result.ts

```diff
@@ -82,7 +83,7 @@ export function buildInteractiveResultCore(
   let longPressTimer: NodeJS.Timeout;
 
   return {
-    select: action,
+    select: debounce(action, options.selectionDelay, {isImmediate: true}),
```

Comment:
The description of `selectionDelay` doesn't imply that it serves a secondary purpose. I think another option should be introduced, such as `selectionDebounce`, or the description should be updated.

Reply:
Good point. I'll add a secondary option. Better that instead of combining two different things.